src/Microsoft.Identity.Web.Sidecar/Microsoft.Identity.Web.Sidecar.csproj

```
@@ -22,6 +22,20 @@
     </NoWarn>
   </PropertyGroup>
 
+  <!-- Trimming + perf (Release only) -->
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
+    <PublishTrimmed>true</PublishTrimmed>
+    <TrimMode>partial</TrimMode>
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
+    <SuppressTrimAnalysisWarnings>false</SuppressTrimAnalysisWarnings>
+    <PublishReadyToRun>true</PublishReadyToRun>
```

Would be interested in image size difference with this vs something like this:

```suggestion
    <SelfContained>true</SelfContained>
```

And then using `mcr.microsoft.com/dotnet/runtime-deps` instead of runtime, would be good to measure later.